src/Raven.Server/Documents/PeriodicBackup/Restore/GoogleCloudRestorePoints.cs

```diff
@@ -58,7 +61,8 @@ protected override ParsedBackupFolderName ParseFolderNameFrom(string path)
         protected override async Task<ZipArchive> GetZipArchive(string filePath)
         {
             Stream downloadObject = _client.DownloadObject(filePath);
-            var file = await RestoreUtils.CopyRemoteStreamLocallyAsync(downloadObject, _configuration);
+            var size = await _client.GetObjectSizeAsync(filePath);
+            var file = await RestoreUtils.CopyRemoteStreamLocallyAsync(downloadObject, size, _configuration, onProgress: null, _cancellationToken);
```

Comment:
what about progress?

Reply:
ping

Reply:
Getting the restore points isn't an operation so it doesn't have a progress.